api/config/services.yaml

```diff
@@ -167,3 +174,27 @@ services:
                 # ensure, data coming from API without timezone information is interpreted as UTC
                 # (e.g. period start '2021-01-01' in POST payload)
                 datetime_timezone: 'UTC'
+
+
+    # override API-platform memory cache configuration
+    # https://github.com/api-platform/core/issues/4975#issuecomment-1253617780
+    api_platform.cache.metadata.property:
+        parent: cache.system
```

Comment:
This is the metadata cache, there's no need need to share this across pods. This config can probably be removed anyway with the next patch version. See here for more details: https://github.com/api-platform/core/issues/4975#issuecomment-1331935076

I think what you mean is HTTP cache as described here: https://api-platform.com/docs/core/performance/
This would of course be a completely different animal and needs sharing across pods.